libtiny_tui/src/msg_area/mod.rs

```diff
@@ -27,9 +28,10 @@ pub(crate) struct MsgArea {
 }
 
 impl MsgArea {
-    pub(crate) fn new(width: i32, height: i32) -> MsgArea {
+    pub(crate) fn new(width: i32, height: i32, scrollback: usize) -> MsgArea {
         MsgArea {
-            lines: Vec::new(),
+            lines: VecDeque::with_capacity(512),
```

Comment:
Thoughts on making this `512.min(scrollback)` ?

Reply:
I think we should do this, yes.